django/views/templates/technical_500.html

```diff
@@ -225,7 +225,9 @@ <h2>Traceback{% if not is_email %} <span class="commands"><a href="#" onclick="r
         </h3></li>
         {% endif %}{% endifchanged %}
         <li class="frame {{ frame.type }}">
-          <code class="fname">{{ frame.filename }}</code>, line {{ frame.lineno }}, in {{ frame.function }}
+          {% if frame.filename %}
```

Comment:
I think we should skip the entire `<li>` when `tb` is `None`. Currently we display an empty frame:

![obraz](https://user-images.githubusercontent.com/2865885/98075401-9af54a80-1e6c-11eb-9d5f-b3f4fd460faf.png)

Reply:
We need to adjust also the `traceback_area` below:

![obraz](https://user-images.githubusercontent.com/2865885/98075596-f58ea680-1e6c-11eb-8f15-f14b977c68e2.png)